packages/passage/src/stimulus-tabs.jsx

```diff
@@ -84,7 +90,7 @@ class StimulusTabs extends React.Component {
                 label={
                   <Purpose purpose="passage-title">
                     <span
-                      dangerouslySetInnerHTML={{ __html: tab.title }}
+                      dangerouslySetInnerHTML={this.createMarkup(tab.title)}
```

Comment:
I would keep it like: { __html: this.parsedText(tab) } here (instead of using a function that returns an object with __html inside )

Reply:
done

Reply:
done